ERP-BACKEND/app/models/project.py

```
@@ -0,0 +1,52 @@
+    assigned_to = Column(Integer, ForeignKey("users.id", ondelete="SET NULL"), nullable=True)
+    due_date = Column(Date, nullable=True)
+    estimated_hours = Column(Numeric(8, 2), nullable=True)
+    actual_hours = Column(Numeric(8, 2), nullable=True)
+    parent_task_id = Column(Integer, ForeignKey("tasks.id", ondelete="SET NULL"), nullable=True)
+    created_at = Column(DateTime(timezone=True), server_default=func.now())
+    updated_at = Column(DateTime(timezone=True), server_default=func.now(), onupdate=func.now())
+
+    # Relationships
+    project = relationship("Project", back_populates="tasks")
+    assigned_user = relationship("User", back_populates="tasks", foreign_keys=[assigned_to])
+    subtasks = relationship("Task", backref="parent", remote_side=[id])
```

Ruff (0.16.1)

[error] Python builtin is shadowed by class attribute `id` from line 35

(A003)